documentation/recommendations.workspace.adoc

```diff
@@ -0,0 +1,722 @@
+= Recommendations
+:user_name: 'Misty Williams'
+:movie_name: 'Matrix'
+:experimental:
+:icon: font
+:img: ./img
+
+++++
+<style type="text/css">
+* {
+  margin-bottom: 0.5em;
+}
+</style>
+++++
+== Personalized Product Recommendations with Neo4j
+
+image::{img}/openmoviegraph.png[float=right]
+
+=== Recommendations
+
+Personalized product recommendations can increase conversions, improve sales rates and provide a better experice for users. 
+In this guide, we'll take a look at how you can generate graph-based real-time personalized product recommendations using a dataset of movies and movie ratings, but these techniques can be applied to many different types of products or content.
+
+=== Graph-Based Recommendations
+
+Generating *personalized recommendations* is one of the most common use cases for a graph database. 
+Some of the main benefits of using graphs to generate recommendations include:
+
+. *Performance* -- index-free adjacency allows for **calculating recommendations in real time**, ensuring the recommendation is always relevant and reflecting up-to-date information.
+. *Data model* -- the labeled property graph model allows for easily combining datasets from multiple sources, allowing enterprises to **unlock value from previously separated data silos.**
+
+image::{img}/title1.png[width=100%]
+
+****
+Data sources:
+
+* http://www.omdbapi.com/[Open Movie Database]
+* https://grouplens.org/datasets/movielens/[MovieLens]
+****
+
+== The Open Movie Graph Data Model
+
+
+=== The Property Graph Model
+
+The data model of graph databases is called the labeled property graph model.
+
+*Nodes*: The entities in the data.
+
+*Labels*: Each node can have one or more *label* that specifies the type of the node.
+
+*Relationships*: Connect two nodes. 
+They have a single direction and type.
+
+*Properties*: Key-value pair properties can be stored on both nodes and relationships.
+
+=== Eliminate Data Silos
+In this use case, we are using graphs to combine data from multiple sources.
+
+*Product Catalog*: Data describing movies comes from the product catalog silo.
+
+*User Purchases / Reviews*: Data on user purchases and reviews comes from the user or transaction source.
+
+By combining these two in the graph, we are able to query across datasets to generate personalized product recommendations.
+
+image::{img}/datamodel.png[width=100%]
+
+=== Nodes
+
+`Movie`, `Actor`, `Director`, `User`, `Genre` are the labels used in this example.
+
+=== Relationships
+
+`ACTED_IN`, `IN_GENRE`, `DIRECTED`, `RATED` are the relationships used in this example.
+
+=== Properties
+
+`title`, `name`, `year`, `rating` are some of the properties used in this example.
+
+
+== Intro To Cypher
```

Comment:
Depending on how @ian-neo4j  wants to do do it we can leave off the intro to cypher here, if it will be available in another place.

Reply:
I'll leave it for now, pending Ian's input.